hack/ci/run-conformance-tests.sh

```diff
@@ -52,7 +52,7 @@ elif [[ $provider == "packet" ]]; then
   EXTRA_ARGS="-packet-api-key=${PACKET_API_KEY}
     -packet-project-id=${PACKET_PROJECT_ID}
     -packet-kkp-datacenter=packet-am"
-elif [[ $provider == "gcp" ]] || [[ $provider == "gce" ]]; then
+elif [[ $provider == "gcp" ]] || [[ $provider == "gcp" ]]; then
```

Comment:
Looks like some leftover of replace function
```suggestion
elif [[ $provider == "gcp" ]]; then
```